environments/icds-cas/app-processes.yml

```diff
@@ -439,56 +217,9 @@ celery_processes:
       pooling: gevent
       concurrency: 10
       num_workers: 2
-  'celery34':
-    reminder_case_update_queue:
-      pooling: gevent
-      concurrency: 5
-      num_workers: 4
-    reminder_queue:
-      pooling: gevent
-      concurrency: 5
-      num_workers: 6
-    sms_queue:
-      pooling: gevent
-      concurrency: 10
-      num_workers: 2
-  'web28':
-    ucr_indicator_queue:
-      concurrency: 3
-  'web29':
-    ucr_indicator_queue:
-      concurrency: 4
-  'web30':
-    ucr_indicator_queue:
-      concurrency: 4
-  'web31':
-    ucr_indicator_queue:
-      concurrency: 4
-  'web32':
-    ucr_indicator_queue:
-      concurrency: 4
-  'web33':
-    ucr_indicator_queue:
-      concurrency: 4
-  'web34':
-    ucr_indicator_queue:
-      concurrency: 4
-  'web35':
-    ucr_indicator_queue:
-      concurrency: 4
-  'web36':
-    ucr_indicator_queue:
-      concurrency: 4
-  'web37':
-    ucr_indicator_queue:
-      concurrency: 4
-  'web38':
-    ucr_indicator_queue:
-      concurrency: 4
-  'web39':
-    ucr_indicator_queue:
-      concurrency: 4
-  'web40':
+    icds_aggregation_queue:
```

Comment:
@calellowitz is it OK to increase the number of process for this queue?

Reply:
This queue won't be needed after today, so I don't think this change is necessary, although i'm not sure its harmful per se. I definitely dont think we need to add it to two more machines at this level

Reply:
Changed the aggregation queue to have it on only single Server.